app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,8 @@
+class PetApplicationsController < ApplicationController
+
+  def index
+    @pet = Pet.find(params[:id])
+    @applications = @pet.applications.all
```

Comment:
Might want to think about this (line 5) being a helper method in the Pet model but can definitely be a refactor later!